frontend/src/components/NavigationBar/index.tsx

```diff
@@ -139,10 +139,10 @@ function NavigationBar() {
                 {user && user.role === "teacher" ? (
                   <MenuItem
                     onClick={() => {
-                      window.location.href = "/templates";
```

Comment:
edit the function such that i open the `/recordings `page in a new tab. only do this for desktop though

may think of using window.open method, check google or documentation on how to implement the new tab thing